ajax/libs/vue-select/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "vue-select",
+  "filename": "vue-select.js",
+  "version": "1.3.3",
+  "description": "A native Vue.js component that provides similar functionality to Select2 without the overhead of jQuery.",
+  "author": "Jeff Sagal <sagalbot@gmail.com>",
+  "homepage": "https://sagalbot.github.io/vue-select/",
+  "license": "MIT",
+  "keywords": [
+    "vue.js",
+    "vue-select",
+    "select2",
+    "tagging"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/sagalbot/vue-select.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/sagalbot/vue-select.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

Comment:
@lcd78706 Let's use `"**/vue-select*"` here because `"**/*"` will fetch the files under `dist` folder we don't need before v1.1.0.